kollect-core/src/main/kotlin/kollect/arrow/instances/effects/instances.kt

```diff
@@ -0,0 +1,16 @@
+package arrow.effects
+
+import arrow.Kind
+import arrow.typeclasses.MonadError
+import kollect.KollectMonadError
+import kollect.Query
+
+object IOKollectMonadErrorInstance : KollectMonadError<ForIO> {
```

Comment:
We need to provide separate modules if possible with instances for `DeferredK`, `ObservableK`, `FlowableK` etc.

Reply:
👍  I can do it in the next PRs